libs/vm_monitor/src/cgroup.rs

```diff
@@ -33,6 +35,7 @@ impl Default for Config {
             memory_poll_interval: Duration::from_millis(100),
             memory_history_len: 5, // use 500ms of history for decision-making
             memory_history_log_interval: 20, // but only log every ~2s (otherwise it's spammy)
+            memory_history_log_noskip_interval: Duration::from_secs(60), // but only if it's changed, or 60 seconds have passed
```

Comment:
let's put this more frequent, at e.g. 15s ? That way it's not lower resolution than our metrics (ext-metrics datasource is 15s IIRC). If it's still a substantial log volume, I'd be happy to reduce it further.

Reply:
My main concern is less that the data isn't available elsewhere, it's more about not needing to cross-compare different datasources. Add onto it that metrics are imprecise (if there's datapoints every 15s, I don't know when the data was actually collected in that 15-second interval), and I've typically found logs to be much more helpful for things like investigating OOMs.

Reply:
> Isn't that implied by the timestamp associated with the collection of the metric?

My understanding is that this is unfortunately not quite the case — for example, if there's data every 15s, you will get datapoints at 00:00:00, 00:00:15, 00:00:30, etc. But they might've been collected at 23:59:53, 00:00:08, 00:00:23, etc.

(again: this is just "as far as I can tell" -- I'm by no means an expert here. A while back I tried to make a compute metrics dashboard that used cAdvisor's `container_last_seen` metric, which _does_ have the actual timestamp used, and it was a bit off from the time associated with the value)